action.yml

```diff
@@ -1,5 +1,5 @@
-name: '@guardian/post-release-action'
-description: 'A GitHub action to handle the post release process for repositories with branch protection'
+name: '@guardian/actions-merge-workflow-changes-to-protected-branch'
+description: 'A GitHub action to open, approve and merge a PR with changes generated during a workflow action on a protected branch'
```

Comment:
This name and description suggest a flexibility and agnosticism (Is that a word? 🤔 ) towards the changes created by the workflow that the action itself does not currently possess. @sndrs and I discussed yesterday taking the expected/accepted changes as an input but I'm still not sure whether that should be through the form of `additionalChanges` where the version changes are still expected or entirely configurable. 

Interested in any thoughts on both the name and description and also on the prospect of opening up the expected changes to more configurability.

Reply:
Just to add to this, we also leave a comment in the PR along the lines of `Updating the version number in the repository following the release of v0.38.1`